### PR TITLE
Add minZoom and maxZoom properties

### PR DESCRIPTION
**Pull request by nicknikolov**
🎥

### CODE REVIEW

**Review by vorg — CHANGES_REQUESTED** [review]
Please use existing minDistance and maxDistance instead of new props like minZoom

File: example/index.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ const camera = createCamera({
 createOrbiter({
   camera: camera,
   element: ctx.gl.canvas,
-  easing: 0.1
+  easing: 0.1,
+  minDistance: 3,
```
Comment (vorg) [inline example/index.js]: isn't that arbitrary and project specific? Most my cameras are positioned at 1,1,1...

File: example/index.js [hunk 2]
```diff
@@ -27,7 +27,9 @@ const camera = createCamera({
...
+  maxDistance: 7
```
Comment (vorg) [inline example/index.js]: as above

Comment (nicknikolov) [reply example/index.js]: Just testing that it works but I'll remove it

**Review by vorg — APPROVED** [review]

**Pull request merged**